src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -77,6 +96,7 @@ TwitchMessageBuilder::TwitchMessageBuilder(
     , action_(_ircMessage->isAction())
 {
     this->usernameColor_ = getApp()->themes->messages.textColors.system;
+    this->highlightSoundUrl_ = getFallbackHighlightSound();
```

Comment:
We shouldn't initialize `highlightSoundUrl_` if we don't need it. If we really need to initialize it in the constructor, I'd rather we just set it to `QUrl("qrc:/sounds/ping2.wav")` instead of making filesystem calls in every constructor.

Reply:
Turns out we don't need this at all. :sweat_smile:

Rsolved in d44d803f.